tests/Silex/Tests/Provider/TwigServiceProviderTest.php

```diff
@@ -82,4 +84,24 @@ public function testLoaderPriority()
         });
         $this->assertEquals('foo', $app['twig.loader']->getSourceContext('foo')->getCode());
     }
+
+    /**
+     * @return \PHPUnit\Framework\MockObject\MockObject
+     */
+    protected function getMockFormFactory()
+    {
+        return $this->getMockBuilder('Symfony\Component\Form\FormFactoryInterface')->getMock();
+    }
+
+    /**
+     * @param \Silex\Application $app
+     *
+     * @return void
+     */
+    protected function setMockFormFactoryForApplication(Application $app): void
```

Comment:
When something is changed we should always return the changed instance to make clear that there it was changed.